AUTHORING_GUIDE.md

```diff
@@ -1,207 +1,117 @@
-1. Never alias imports unless there is a name collision.
-1. Use `.format()` over `%` for string interpolation.
-1. Generally put a blank line above control statements that start new indented
-   blocks, for example:
-
-    ```python
-    # Good
-    do_stuff()
-
-    if other_stuff():
-        more_stuff()
-
-    # Not so good
-    do_stuff()
-    if other_stuff():
-        more_stuff()
-    ```
-
-   This rule can be relaxed for counter or accumulation variables used in loops.
-1. Don't use parentheses on multiple return values (`return one, two`) or in
-   destructuring assignment (`one, two = some_function()`).
-2. Prefer not to do hanging indents if possible. If you break at the first
-   logical grouping, it shouldn't be necessary. For example:
-
-    ```python
-    # Good
-    do_some_stuff_please(
-        a_parameter, another_parameter, wow_so_many_parameters,
-        much_parameter, very_function)
-
-    # Not so good
-    do_some_stuff_please(a_parameter, another_parameter, wow_so_many_parameters,
-                         much_parameter, very_function)
-    ```
-
-   Similarly with strings and other such things:
-
-    ```python
-    long_string = (
-        'Alice was beginning to get very tired of sitting by her sister on '
-        'the bank, and of having nothing to do: once or twice she had peeped '
-        'into the book her sister was reading, but it had no pictures or ...'
-    )
-    ```
-1. Use descriptive variables names in comprehensions, for example:
-
-    ```python
-    # Good
-    blogs = [blog for blog in bob_laws_law_blog]
-
-    # Not so good
-    blogs = [x for x in bob_laws_law_blog]
-    ```
-
-## The sample format
-
-In general our sample format follows ideas borrowed from
-[Literate Programming](https://en.wikipedia.org/wiki/Literate_programming).
-Notably, your sample program should be self-contained, readable from top to bottom,
-and fairly self-documenting. Prefer descriptive names. Use comments
-and docstrings only as needed to further explain. Always introduce functions and
-variables before they are used. Prefer less indirection. Prefer imperative
-programming as it is easier to understand.
-
-### Shebang
+We're happy you want to write a Python sample! Like a lot of Pythonistas,
+we're opinationed and fussy. This guide is a reference for the format and
+style expected of samples contributed to the
+[python-docs-samples](https://github.com/GoogleCloudPlatform/python-docs-samples)
+repo. The guidelines below are intended to ensure that all Python samples
+meet the following goals:
+
+* **Copy-paste-runnable.** A developer should be able to copy and paste the
+code into their own environment and run it with as few modifications as
+possible.
+* **Teach through code.** Each sample should demonstrate best practices for
+interacting with Google Cloud libraries, APIs, or services.
+* **Idiomatic.** Each sample should follow widely accepted Python best
+practices as covered below.
+
+# Sample Guidelines
```

Comment:
nit: Usually a single `#` indicates title, and then headers are `##` or more. Not sure if this matters for Github's rendering (as it looks fine as is), but may cause issues later if we decide to autogen a table of contents or something.

Reply:
Addressed in commit [30673e7](https://github.com/GoogleCloudPlatform/python-docs-samples/pull/2728/commits/30673e7853619a8bbf193a93eb41c9a65061303d).